documentation/cluster-controller.adoc

```diff
@@ -224,6 +227,34 @@ configuration file. For this reason, you can find more information on how to use
 
 For more information on how metrics work, the related documentation is available link:../../metrics/METRICS.md[here]
 
+===== Topic controller
```

Comment:
This is good. But I think the Topic Controller documentation should probably contain at least a short mention that it can be deployed directly with the cluster.

Reply:
@scholzj there is no mentions for the other ways to deploy it as well. Is it better having @tombentley updating the topic controller doc for that ?